# Passing device_id to safety server is necessary for control server to identify the place where emergency occurred
string[] device_id_vec

# Register EmergencyScore to safety server.
SurroundEmergency[] emergency_scores

---
# response is true if value accepted, otherwise server ignore the request and response false.
bool response